### DISCUSSION TITLE
DCL not set according to utils.py

### DISCUSSION THREAD

**Discussion by hansgeorgk** [Louisvdw/dbus-serialbattery#345] [Q&A] [answered]
Hi,
thanks a lot for this new release - very much appreciated.

just installed v141 and seems like the current values are stuck with the defaults ...
adjusted utils.py:
# battery Current limits
MAX_BATTERY_CHARGE_CURRENT = 100.0
MAX_BATTERY_DISCHARGE_CURRENT = 200.0

and rebooted my EasySolar II GX device

but I get this now in the console
<img width="500" alt="Bildschirmfoto 2022-12-12 um 14 06 59" src="https://user-images.githubusercontent.com/100286224/207053204-eb87f9f7-e0bc-425a-905e-3d95e1080402.png">

ok, for Tage 5A CCL as I am at 99% :-)
But DCL should be 200 according to utils.py ...

anything else needs to be changed in v141?

thanks
cheers
HG

**Comment by Louisvdw** [comment, marked as answer]
Yes there are 3 attributes that can limit. If you don't disable the 3x CCCM_ and 3x DCCM_ then those limits also apply, so you need to adjust them as well. (Still need to update the wiki with all the details)

See #331 for the list.
